### ISSUE TITLE
[Quality of life] hole under rock, Ancestral Plans

### ISSUE THREAD

**Issue by BoB-Client** [Thy-Vipe/BeastsOfBermuda-issues#3422]
_Originally written by **disaster strikes | 76561198040981157**_
Game Version: 1.1.1085

*===== System Specs =====
CPU Brand: Intel(R) Core(TM) i7-9700K CPU @ 3.60GHz
Vendor: GenuineIntel
GPU Brand: NVIDIA GeForce RTX 2070 SUPER
GPU Driver Info: Unknown
Num CPU Cores: 8
===================*
Context: **ory can access it, which im playing atm lol** 
Map: Ancestral_Plains

Location: X=-37057.730 Y=-32010.016 Z=8698.090

simply put there is a nice hole under this rock at this location that an ory and maybe a velo could fit under

**Comment by Predatoria** [comment]
Deleted the rock in question
